corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -150,6 +153,21 @@ hqDefine("app_manager/js/details/screen", function () {
             });
         };
 
+        // Given a column model, return a boolean indicating whether the column is on an odd
+        // or an even tab, for the sake of being able to differentiate in the case tile preview
+        // which rows go with which tab.
+        self.tabPolarity = function (column) {
+            var self = this,
+                flag = false;
+            _.find(self.columns(), function (c, i) {
```

Comment:
I could not verify that `_.find` is guarantied to run in over the items in sequence but since JS is single threaded I think this should be the case

Reply:
Interesting, I hadn't thought about that. I suppose `each` would guarantee the order, though it also wouldn't end early.

Reply:
Although, looking at the [docs](https://underscorejs.org/#find), I think "returning the first one that passes a truth test" implies that the items are examined in order.